conf/modules.config

```diff
@@ -134,13 +134,14 @@ process {
             enabled: params.run_protein_annotation,
             saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
         ]
-        ext.args   = [
+                // Warning: Do not disable the flags "--enable-tsv-residue-annot" and "--formats tsv"! This would cause a run failure because the format of the resulting files would no longer be adequate for parsing by AMPcombi2.
```

Comment:
```suggestion
            // Warning: Do not disable the flags "--enable-tsv-residue-annot" and "--formats tsv"! This would cause a run failure because the format of the resulting files would no longer be adequate for parsing by AMPcombi2.
```